scripts/production-pull-request-template.sh

```diff
@@ -25,8 +25,9 @@ Reviewing code for this Pull Request is not practical, however, you are asked to
 ## Purchase URL
 * [ ] From the dashboard, create a purchase URL (redirect URL does not matter)
 * [ ] In the purchase URL, configure metadata collection to collect an email address (\`\"metadataInputs\": [ { name: 'Name', type: 'text', required: true } ],\`)
-* [ ] By using a v10 lock, enable multiple purchase in the URL (\`\"maxRecipients\": 6\`)
+* [ ] By using a v12 lock, enable multiple purchase in the URL (\`\"maxRecipients\": 6\`)
 * [ ] Go through the checkout flow and ensure that it works as expected (data is collected for 2 users + and 2 keys are minted)
+* [ ] Use a v9 lock and check if multiple purchase is successful for compatibility. 
```

Comment:
Good idea. We should try to deploy one on Goerli with a short expiration and just point to this one here? wdyt?

Reply:
Yeah, we can add an address / config and change it often as we deprecate the lock version.

Reply:
It we can just upgrade the lock? Want to create one and add all of us as lock managers?

Reply:
Added the unlock gnosis lock which we use on our website as well. 

We don't have contracts deployed to test networks for v9 so need to use a real network. Since we use gnosis one ourselves, we can test it and upgrade it.